package.json

```
@@ -21,7 +21,8 @@
     "clean": "rimraf dist coverage",
     "reinstall": "rimraf flow-typed && rimraf node_modules && npm install && flow-typed install",
     "debug": "cross-env NODE_ENV=debug",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "build-logos": "babel-node --config-file './scripts/babel.config.json' --ignore ./fake --plugins=transform-es2015-modules-commonjs ./scripts/buildLogos.js"
```

What does this `--ignore ./fake` part do? Can we remove it?
```suggestion
    "build-logos": "babel-node --config-file './scripts/babel.config.json' --plugins=transform-es2015-modules-commonjs ./scripts/buildLogos.js"
```

---

Added an ignore flag to fix an issue with babel-node:

```
/Users/dustijones/code/public/paypal-sdk-logos/node_modules/@krakenjs/belter/src/index.js:3
export * from "./device";
^^^^^^
```

I'll see if I can find a long-term solution

---

Looks like it may be a bug that occurs with `babel-node` when you use the `--config-file` flag.  

https://github.com/babel/babel/issues/11892

---

Good find. We can keep this. I just wanted to understand why it is needed.